rq/registry.py

```diff
@@ -268,6 +270,12 @@ def cleanup(self, timestamp: Optional[float] = None):
                         self.death_penalty_class, AbandonedJobError, AbandonedJobError(), traceback.extract_stack()
                     )
 
+                    # we did a job fetch so the meta is supposed to be there
+                    if job.meta:
+                        if ret := job.meta.get('_retried_after_abandonned', 0) < MAX_RETRIES_AFTER_ABANDONED:
+                            job.meta['_retried_after_abandonned'] = ret + 1
+                            self.retries_left = getattr(job, 'retries_left', 0) + 1
```

Comment:
why `self.` here? Don't you want `job.retries_left =`

Reply:
Because I'm stupid? 😬